kubetest2-gke/deployer/up.go

```diff
@@ -264,7 +271,7 @@ func generateClusterNames(numClusters int, uid string) []string {
 
 func validateVersion(version string) error {
 	switch version {
-	case "latest":
+	case "latest", "":
```

Comment:
wanted to avoid this, but looks like the default version in the release channel is not the same as latest which is a different behavior from not using a release channel.

Let's also change the default flag value to empty string then.

Reply:
Do you want me to change the default flag value to empty string for release channel only, or both? 

I did some quick experiment, the cluster version for `gcloud container clusters create test1 --cluster-version=` and `gcloud container clusters create test2 --cluster-version=latest` is actually different, the former is `1.16.13-gke.401` and the latter is `1.17.12-gke.500`.

Reply:
Let's default to empty string for both cases.

Reply:
Done.